lib/spack/spack/relocate.py

```diff
@@ -651,8 +651,7 @@ def is_relocatable(spec):
     return True
 
 
-def file_is_relocatable(
-        file, paths_to_relocate=[spack.store.layout.root, spack.paths.prefix]):
+def file_is_relocatable(file):
```

Comment:
This change will break binary relocation.

If the issue is lists as default arguments, we should change this to `def file_is_relocatable(file, paths_to_relocate=None)` and insert the defaults below if we see `paths_to_relocate is None`

Reply:
@becker33, see latest change.